one-step publishing

blackmad opened the pull request.

jsani-radar commented on package.json:
```diff
@@ -20,7 +20,8 @@
     "build:iife-min": "rollup -c rollup-iife-min.config.js",
-    "report-coverage": "./node_modules/.bin/codecov"
+    "report-coverage": "./node_modules/.bin/codecov",
+    "postpublish": "PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]') echo VERSION: $PACKAGE_VERSION && git tag $PACKAGE_VERSION && git push --tags &&  for i in dist/*; do aws s3 cp $i s3://js.radar.io/$PACKAGE_VERSION/; done && echo do not forget to update README"
```
not sure if the trailing `%` would end up causing problems:
<img width="953" alt="Screen Shot 2020-07-01 at 1 45 52 PM" src="https://user-images.githubusercontent.com/58698757/86275388-32861c00-bba1-11ea-8568-e5083ed2e2f4.png">

kochis replied:
```
cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]'
```
Can we just use `jq` for this?
```
cat package.json | jq .version
```

kochis commented:
```
for i in dist/*;
```
I don't know if we want `index.js` in the public folder.

kochis commented:
```
cp $i s3://js.radar.io/$PACKAGE_VERSION/
```
Should be prefixed with `v`
```
cp $i s3://js.radar.io/v$PACKAGE_VERSION/

blackmad commented:
changed to vXXX, changed to jq, not uploading index.js

On Wed, Jul 1, 2020 at 1:59 PM Craig Kochis <notifications@github.com>
wrote:

> cp $i s3://js.radar.io/$PACKAGE_VERSION/
>
> Should be prefixed with v
>
> cp $i s3://js.radar.io/v$PACKAGE_VERSION/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/radarlabs/radar-sdk-js/pull/32#issuecomment-652565201>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADMZMF42IJLIYOZ52YFLX3RZN2ONANCNFSM4ONUZNXA>
> .
>

jsani-radar commented on package-lock.json:
```diff
@@ -5026,7 +5026,7 @@
       "requires": {
         "@sinonjs/formatio": "3.0.0",
-        "just-extend": "^4.0.0",
+        "just-extend": "^3.0.0",
```
I don't think we need any `package-lock.json` updates for an additional `npm` script - @kochis unfortunately I think we're gonna keep running into this 😬
